src/external/CMakeLists.txt

```diff
@@ -52,6 +52,15 @@ add_library(sdl2 INTERFACE)
 target_include_directories(sdl2 INTERFACE "${SDL2_INCLUDE_DIR}")
 target_link_libraries(sdl2 INTERFACE "${SDL2_LIBRARY}")
 
+if(WIN32)
+	add_library(wsock32 INTERFACE)
+	find_library(WSOCK32 wsock32)
+	target_link_libraries(wsock32 INTERFACE WSOCK32)
+	add_library(ws2_32 INTERFACE)
+	find_library(WS2_32 ws2_32)
+	target_link_libraries(ws2_32 INTERFACE WS2_32)
```

Comment:
I'd probably combine both of these into the same library.

Do we need both anyway? Can we just use Winsock 1 or 2 but not both?